openreview/stages/venue_stages.py

```diff
@@ -539,7 +539,9 @@ def __init__(self,
         remove_fields = [],
         rating_field_name = 'rating',
         confidence_field_name = 'confidence',
-        process_path = None
+        process_path = None,
+        content_query = {},
```

Comment:
I'm not sure about calling `content_query`, it doesn't indicate that is the source of submissions that we want to create the review invitations.

what about:

- selected_submissions
- selected_submissions_query
- selected_submissions_content_query
- forums_query
- selected_forums
- replyto_query
...